learn/app-development/widgets/datalive/form/button.md

```diff
@@ -15,17 +15,17 @@ The button widget can trigger any of the following actions when it is clicked.
 
```

Comment:
This page does not have sidebar/leftnav

Reply:
This page also does not have leftnav learn/app-development/widgets/use-cases-prefabs

Reply:
Added the side nav for prefab use cases. 
Deleted the duplicate files for the button documentation.